docs/README.rst

```diff
@@ -124,6 +124,8 @@ Define the parameters in the ``env`` section of your `application manifest </con
 +----+-------------------------------------+---------+----------+----------------+---------------------------------------------------------------------------------+----------------+
 |    | health_monitors                     | array   | Optional | n/a            | Health monitors attached to each configured routing pool                        |                |
 +----+-------------------------------------+---------+----------+----------------+---------------------------------------------------------------------------------+----------------+
+|    | session_persistence                 | boolean | Optional | true           | Enables JSESSIONID cookie session persistence on the BIG-IP                     | true, false    |
```

Comment:
This lives outside the BIG-IP section in the config so this should be moved down by route_mode

Reply:
Sounds good I'll move it.